### ISSUE TITLE
Language selector navigation button missing new `aria-haspopup` attribute

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt-contrib-languagePicker#108]
### Subject of the issue
The core has recently been updated to add a `aria-haspopup` attribute with the value of `dialog` to the navigational elements that open a drawer overlay. 

Please refer to the [core issue](https://github.com/adaptlearning/adapt-contrib-core/issues/657) for the research and discussion.
